### PR TITLE
Browse insights and Add new insight

### PR DESCRIPTION
**Pull request by mk4111**
Linked to #324 

Allows users to browse insights, and add the:
- title
- url
- author 
- document type
- resource (tickbox)
of a new insight.

`Browse` and `Add` have been placed in the same PR for testing purposes

### CODE REVIEW

File: server/models/add-insight.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+var Joi = require('joi');
+var addInsightView = require('../handlers/insights/add-view.js');
+
+module.exports = {
+  payload: {
+    title: Joi.string().min(1).required(),
```
Comment (JMurphyWeb) [inline server/models/add-insight.js]: these need max's as well

File: templates/views/insights/add.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      <a class="icon icon--cancel" href="/orgs/{{permissions.organisation_id}}"></a>
+    </div>
+
+    {{#if error}}
+    <div class="snackbar snackbar-error">
+      <h1 class="error">{{error.message}}</h1>
+    </div>
+    {{else}}
+    <div class="snackbar snackbar-success">
+      <h1 class="error">Have you uploaded your asset? If not do it [here](ADD LINK)</h1>
```
Comment (JMurphyWeb) [inline templates/views/insights/add.html]: need to add remove the add link mark down

**Comment by JMurphyWeb** [comment]
looks good. A few small things that can be changed.

doctype needs to be cleared up/ replaced with `type`

**Comment by SimonLab** [comment]
@JMurphyWeb the coverage is down, do we have missing tests?

**Comment by JMurphyWeb** [comment]
@SimonLab I believe this is due to the strange order of the PRs

**Comment by JMurphyWeb** [comment]
@SimonLab I will do this as soon as insights PRs are merged: https://github.com/emfoundation/ce100-app/issues/519

**Comment by JMurphyWeb** [comment]
Just checked and it is ok to merge this now. the lines are covered in a later pr.

**Comment by SimonLab** [comment]
@JMurphyWeb Thanks for confirming that the tests are added on another PR

File: package.json [hunk 3]
```diff
@@ -42,6 +42,7 @@
     "joi": "^9.0.4",
     "jsonwebtoken": "^7.1.9",
     "pg": "^6.1.0",
+    "pg-insights": "0.0.6",
```
Comment (SimonLab) [inline package.json]: The last version is 0.0.7

File: server/handlers/insights/add-view.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      name: type
+    };
+  });
+  var options = Object.assign(
+    { doctype: docDropdown },
+    permissions,
+    { error: error }
+  );
+
+  if (loggedIn.scope !== 'admin') {
```
Comment (SimonLab) [inline server/handlers/insights/add-view.js]: we could check if the user is admin before creating the options object

**Pull request merged**